### PR TITLE
add Post.editedAt ("edited_at")

### PR DESCRIPTION
**Pull request by technicat**
Added relative to other Post properties same as in the API doc:

https://docs.joinmastodon.org/entities/Status/#edited_at

### CODE REVIEW

File: Tests/TootSDKTests/PostTests.swift [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        XCTAssertEqual(result.id, "112300188527054704")
+        XCTAssertEqual(result.filtered, [])
+        XCTAssertEqual(result.mentions, [])
+        XCTAssertEqual(result.emojis, [])
+        XCTAssertEqual(result.mediaAttachments, [])
+        XCTAssertEqual(result.tags.count, 1)
+        XCTAssertEqual(result.favourited, false)
+        XCTAssertEqual(result.reposted, false)
+        XCTAssertNotNil(result.createdAt)
+        XCTAssertNotNil(result.editedAt)
```
Comment (kkostov) [inline Tests/TootSDKTests/PostTests.swift:29]: Could you add an equality check as well, just to make sure the correct time and tz are parsed.

Comment (technicat) [reply Tests/TootSDKTests/PostTests.swift:29]: Is there an example of a date check in the tests? I don't see any.

Comment (kkostov) [reply Tests/TootSDKTests/PostTests.swift:29]: We have some e.g. https://github.com/TootSDK/TootSDK/blob/main/Tests/TootSDKTests/ActivityTests.swift

**Comment by kkostov** [comment]
@technicat Thank you for contributing this. Did you see something that would prevent it from being merged? I thought it was pretty cool to add.

**Comment by technicat** [comment]
I just didn't want to spend any more time on the test.

**Comment by kkostov** [comment]
I hear you, is it ok if I add the gets and merge it?

**Comment by technicat** [comment]
sure!

**Comment by kkostov** [comment]
Awesome, thanks!

**Pull request closed without merging**